### PR TITLE
Delete all workspaces added in the workspaces hamburger

### PR DESCRIPTION
**Pull request by LianaHus**
fix https://github.com/ethereum/remix-project/issues/3379

### CODE REVIEW

**Comment by ioedeveloper** [comment]
I reviewed the code and it looks good to me.

File: apps/remix-ide/src/app/tabs/locales/zh/filePanel.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
   "filePanel.workspace.rename": "重命名工作空间",
   "filePanel.workspace.delete": "删除工作空间",
   "filePanel.workspace.deleteConfirm": "确定要删除当前工作空间?",
+  "filePanel.workspace.deleteAll": "Delete All Workspaces",
+  "filePanel.workspace.deleteAllConfirm": "Are you absolutely sure you want to delete all your workspace?",
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/tabs/locales/zh/filePanel.json:13]: I think it should be `all your workspaces`

Comment (Aniket-Engg) [reply apps/remix-ide/src/app/tabs/locales/zh/filePanel.json:13]: Not sure if we should also add something like `Deleted workspaces can not be restored in any manner`

Comment (LianaHus) [reply apps/remix-ide/src/app/tabs/locales/zh/filePanel.json:13]: thanks. will update

File: apps/remix-ide/src/app/tabs/locales/en/filePanel.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
   "filePanel.workspace.rename": "Rename Workspace",
   "filePanel.workspace.delete": "Delete Workspace",
   "filePanel.workspace.deleteConfirm": "Are you sure to delete the current workspace?",
+  "filePanel.workspace.deleteAll": "Delete All Workspaces",
+  "filePanel.workspace.deleteAllConfirm": "Are you absolutely sure you want to delete all your workspace? Deleted workspaces can not be restored in any manner.",
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/tabs/locales/en/filePanel.json:13]: Can you make `all your workspace` to `all your workspaces`

**Review by Aniket-Engg — APPROVED** [review]

**Review by bunsenstraat — CHANGES_REQUESTED** [review]
The two next buttons after delete all have lost their hover and don't seem to be working in CI either.

**Review by bunsenstraat — APPROVED** [review]

**Pull request merged**